gui/src/components/TopBar.tsx

```diff
@@ -36,7 +36,7 @@ export function TopBar({
             className="mx-2 flex justify-around flex-col text-standard-bold text-status-success bg-status-success bg-opacity-20 rounded-lg px-3"
             data-tauri-drag-region
           >
-            v{packagejson.version}
+            v{packagejson.version}-{window.__COMMIT_HASH__}
```

Comment:
Do we want a commit hash to be shown for stable releases? I'd like to get some other opinions on this.

Reply:
If its tagged in git, it should use the tag. If its not tagged, it should use the hash.
We could *also* include the hash in the development view though.